daemon/src/peer/run_state/running_waiting_room.rs

```diff
@@ -328,7 +328,7 @@ impl RunningWaitingRoom {
             },
             // FIXME(alexjg): Figure out how to report this error to the client
             Err(error) => {
-                log::warn!("WaitingRoom::Error : {}", error);
+                tracing::warn!(?error, "waiting room error");
```

Comment:
Curious as to why you use `?` instead of `%` in a lot of places?

Reply:
Usually the `Debug` implementation provides more information about the printed value. For errors in particular using `Display` may omit information like constructor names if `#[error(transparent)]` is used or any field that is not explicitly mentioned in the error message. Of, course for URN it makes more sense to use `%`.